src/ripple/protocol/impl/SField.cpp

```diff
@@ -91,6 +91,7 @@ CONSTRUCT_UNTYPED_SFIELD(sfMetadata,            "Metadata",             METADATA
 CONSTRUCT_TYPED_SFIELD(sfCloseResolution,       "CloseResolution",      UINT8,      1);
 CONSTRUCT_TYPED_SFIELD(sfMethod,                "Method",               UINT8,      2);
 CONSTRUCT_TYPED_SFIELD(sfTransactionResult,     "TransactionResult",    UINT8,      3);
+CONSTRUCT_TYPED_SFIELD(sfScale,                 "Scale",                UINT8,      4);
```

Comment:
The integer value at the end of this macro (`4`) is filled into the `SField::fv` (`field value`). Is there a reason why you chose 4? Or do they just need to be unique? I'm trying to understand more about these values.

Reply:
It has to be unique. Selecting next in sequence makes it easier for the next update to see what's available.